### ISSUE TITLE
How to use a CapabilitiesServer from the french authority IGNF

### ISSUE THREAD

**Issue by TimThaler** [BruTile/BruTile#179]
we are trying to use an server from IGN France.
Some of these nodes use the 
```
<TileMatrixSet>
  <ows:Identifier>LAMB93_1M_MNT</ows:Identifier>
  <ows:SupportedCRS>IGNF:LAMB93</ows:SupportedCRS>
  <TileMatrix>
```
Also the TileSet is missing the `ScaleDenominator` and the `TopLeftCorner`. 
Is it possible to configure Brutile in a way to make use of this server?

**Comment by pauldendulk** [comment]
Is it a public server, if so do you have a url to the capabilities?

**Comment by TimThaler** [comment]
Hello yes it is - here is the [URL](https://wxs.ign.fr/essentiels/geoportail/wmts?SERVICE=WMTS&REQUEST=GetCapabilities&VERSION=1.0.0&LAYER=ORTHOIMAGERY.ORTHOPHOTOS&TILEMATRIXSET=PM)

**Comment by pauldendulk** [comment]
There was an issue with IGNF:LAMB93 which was fixed in #183. It should be fixed in 5.0.3.

**Comment by TimThaler** [comment]
Its working - thanks a lot for that!
